modules/ROOT/pages/managing-instances/backup-restore-export.adoc

```diff
@@ -1,72 +1,157 @@
+See <<snapshot-details>> for more information.
 ====
 
+Aura stores the data securely in encrypted and dedicated cloud storage buckets.
+Backups are stored in the same Cloud Service Provider and region as the instance they are associated with.
+
 == Snapshot types
 
 === Scheduled
 
 label:AuraDB-Professional[]
-label:AuraDB-Enterprise[]
+label:AuraDB-Business-Critical[]
+label:AuraDB-Virtual-Dedicated-Cloud[]
+label:AuraDS-Professional[]
+label:AuraDS-Enterprise[]
 
-A *Scheduled* snapshot is a snapshot that is automatically triggered when you first create an instance, when changes to the underlying system occur (for example, a new patch release), and at a cadence depending on your plan type.
+A *Scheduled* snapshot is a snapshot that is automatically triggered at a cadence depending on your plan type.
 
-Scheduled snapshots are run automatically once a day for Professional instances and once an hour for Business Critical and Enterprise instances.
-
-[NOTE]
-====
-For AuraDB Enterprise database instances running Neo4j v4.x, from day 0 to 7 scheduled snapshots run automatically once every 6 hours.
-From day 8 to 60, snapshots run once a day.
-====
+For Professional instances, scheduled snapshots are run automatically once a day, same for AuraDS instances, and for Business Critical and Virtual Dedicated Cloud instances, they run once an hour.
+See <<snapshot-details>> further on for information about frequency, restorability, and exportability of scheduled snapshots.
 
 === On Demand
 
-An *On Demand* snapshot is a snapshot that you manually trigger by selecting *Take snapshot* from the *Snapshots* tab of an instance.
+An *On Demand* snapshot is a snapshot that you manually trigger with the *Take snapshot* button.
 
 == Snapshot actions
 
-=== Restore
-
-[CAUTION]
-====
-Restoring a snapshot overwrites the data in your instance, replacing it with the data contained in the snapshot.
-====
-
-You can restore data in your instance to a previous snapshot by selecting *Restore* next to the snapshot you want to restore.
-
-Restoring a snapshot requires you to confirm the action by typing RESTORE and selecting *Restore*.
+image::snapshot-actions.png[width=400]
 
+[[export-create]]
 === Export/Create
 
-By selecting the ellipses (*...*) button next to an existing snapshot, you can:
+From the more (*...*) menu next to an existing snapshot, you can:
 
-* *Export* - Download the instance as *_.dump_* file, allowing you to store a local copy and work on your data offline.
+* *Export* - Download the instance as *_.backup_* file, allowing you to store a local copy and work on your data offline.
 * *Create instance from snapshot* - Create a new AuraDB instance using the data from the snapshot.
 
 [NOTE]
 ====
-The ability to Export or Create an instance from a Scheduled Enterprise snapshot is limited to 14 days.
+The ability to Export or Create an instance from a Scheduled Virtual Dedicated Cloud snapshot is limited to 14 days.
```

Comment:
For 4.x this is true, but for 5.x/CalVer this is longer because we're now using backups for this use case, not exports, and for scheduled full backups the number is 60 days (diffs are not exporetable/restorable), so something like this:

```suggestion
The ability to Export or Create an instance from a Scheduled Virtual Dedicated Cloud snapshot is limited to 14 days for version 4.x. For 5.x/CalVer it is limited to 60 days for full scheduled backups, while diff backups are not restorable/exportable and cannot be used to create a new instance from.
```

Reply:
Fixed!